packages/nimble-components/src/wafer-map/template.ts

```diff
@@ -1,18 +1,17 @@
-import { html, repeat } from '@microsoft/fast-element';
+import { html, ref } from '@microsoft/fast-element';
 import type { WaferMap } from '.';
-import type { WaferMapDie } from './types';
 
-// prettier-ignore
 export const template = html<WaferMap>`
-    <div>
-        <ul>
-            ${repeat(
-        x => x.dies,
-        html<WaferMapDie>` <li>
-                    ${x => html<WaferMapDie>`dieX: ${x.x}, dieY: ${x.y}, value:
-                        ${x.value}%`}
-                </li>`
-    )}
-        </ul>
+    <div class="wafer-map-container">
+        <svg class="svg-root ${x => x.orientation}">
+            <g class="zoom-container"}>               
+                <svg class="circle-base " version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewbox="1 .45 20 21">
```

Comment:
I don't know much about SVG rendering but I just want to double check that all of this is necessary. In particular I wonder if we can get rid of:
 - id: I don't see it being referred to from elsewhere
 - xmlns and xmlns:xlink: should these be moved to the root svg element or removed entirely?
 - there also seems to be an extra `}` on the `<g class="zoom-container">` line above

Reply:
Id and links remained from initial implementation. Removed it as requested.
Also, removed the extra `}`